[FIX] Resolve current issues

akimbler opened the pull request:
FIX: Added Data to .gitignore
FIX: Fixed instruction text

akimbler committed:
[FIX] Resolve current issues

FIX: Added Data to .gitignore
FIX: Fixed instruction text

akimbler committed:
Update math_task.py

tsalo commented on math_task.py:
```diff
@@ -96,14 +96,16 @@
     instruction_text = \
     ' You will be shown a series of formulas, \
-    \nyou must determine if the result is less than, equal to,  or greater than 5 \
+    \nyou must determine if the result is less than, equal to, greater than, \
```
Needs "or" and you can drop the last comma.

tsalo committed:
Restructure config file (#8)

* Restructure config file

- Add session level
- Reduce number of trials to 5 (not set to match desired duration yet
of course)
- Use dictionaries instead of lists for storing trial info in config
file
- Set comparison values to be close to solutions
- Rename a/n to analog/numeric.
- Reduce number of runs to 2

* Fix.
